src/istio/oscal-component.yaml

```diff
@@ -1,671 +1,1395 @@
-# add the descriptions inline
 component-definition:
-  uuid: cc873a43-e9fa-433b-8c20-222d733daf1e
-  metadata:
-    title: Istio Controlplane
-    last-modified: "2024-01-18T16:41:56Z"
-    version: "20240118"
-    oscal-version: 1.1.1
-    parties:
-      - uuid: f3cf70f8-ba44-4e55-9ea3-389ef24847d3
-        type: organization
-        name: Defense Unicorns
-        links:
-          - href: https://defenseunicorns.com
-            rel: website
+  back-matter:
+    resources:
+      - rlinks:
+          - href: https://github.com/istio/istio/
+        title: Istio Operator
+        uuid: 60826461-D279-468C-9E4B-614FAC44A306
+      - description: |
```

Comment:
If we intend for the source of truth for this to be maintained outside of this repo I would love if we were:
* remotely referencing these validations
* tying this to a released version of the validation

Some of that might involve questions about sacrificing consuming this in the airgap but if I'm understanding all of the pieces correctly we could perhaps run `lula tools compose` in CI and capture that as some kind of release artifact.

Reply:
great feedback - I'd like to consider what the implications involved here are as maybe beyond scope of this first iteration. 

I agree fully that we want to tie a given artifact back to a versioned source. We need to re-work our expectations around where OSCAL lives. Current discussion we've been having is removing the middle-man - publishing OSCAL in target locations (uds-core for instance) and then having `compliance-artifacts` be solely for validations. If we do this then yes I agree if others want to introduce "composing" later in the lifecycle.

Reply:
I think I agree with the sentiment here - current state would make it hard for me as a dev on uds-core to easily identify source of truth for validations/oscal and know where to contribute meaningfully.

I don't think I'm opposed to the source of truth being external, but it would be great if that were referenced here (and versioned as Rob mentioned). I'm okay with this as the first iteration, I think it falls in the same bucket as "how do I identify/triage failures" - it's workable but not ideal. Open to other takes though - probably should be tracked in an issue for further discussion/working if we do punt on it (unsure if that should be on core or compliance repo).

Reply:
Added as additional context in a current release issue.

Reply:
Other additional context here - we are adding a constraint on network connectivity for every execution (3 flavors) for every validation we develop. These will be growing over-time but we have not stress-tested what this looks like or the overlay of github runners network reliability. 

We can buy down on some of that if we performed a compose in CI -> uploaded the artifact -> and then downloaded the composed version for each execution. More CI steps but we may very well be producing the composed artifact at some point anyway..

Reply:
@brandtkeller Can you link to the issue?

Reply:
Current placeholder [issue](https://github.com/defenseunicorns/compliance-artifacts/issues/51) for that work with additional thoughts and links.